.github/workflows/build-agent.yml

```diff
@@ -0,0 +1,95 @@
+name: Build Agent
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - v*
+    paths:
+    - 'agent/**'
+
+  pull_request:
+    branches: [ master ]
+    paths:
+    - 'agent/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Define a fixed version for your build image or expect that your build will break at some point - been there done that!

Reply:
You mean due to underlying changes on ubuntu image? makes sense.

@gomex can you fix this to 18.04 for now?

Reply:
Yes, or sometimes GitHub/GitLab/etc defines different package that are included in the `latest` release

Reply:
I agree ;) I didn't change it yet, but it was in my plan to request it.